R/create_mbd_params.R

```diff
@@ -0,0 +1,25 @@
+#' Create mbd parameters
+#' @inheritParams default_params_doc
+#' @author Richel J.C. Bilderbeek, Giovanni Laudanno
+#' @export
+create_mbd_params <- function(
+  lambda,
+  mu,
+  nu,
+  q,
+  cond = 1,
```

Comment:
I hope we can prevent using numbers instead of words. I'd prefer a string like `non_extinction` or whatever.

Reply:
mmm, not sure